cddl/tst-info.cddl

```diff
@@ -1,32 +1,28 @@
 TST-info-based-on-CBOR-time-tag = {
-  &(version : 0) => int .default 1 ; obsolete?
+  &(version : 0) => int .default 1
   &(policy : 1) => oid
   &(messageImprint : 2) => MessageImprint
   &(serialNumber : 3) => int
   &(eTime : 4) => profiled-etime
-  &(ordering : 5) => bool .default false
+  ? &(ordering : 5) => bool .default false
   ? &(nonce : 6) => int
   ? &(tsa : 7) => GeneralName
   * $$TSTInfoExtensions
```

Comment:
Do you want extensions to be ignored when unknown (you would need to add a catch-all) or fail on unknown (in which case what we have is fine)?

Reply:
the extensions here MUST match an extension in the corresponding request.  Which bears the bigger question: is this CBOR tst-info something that we can define in isolation?  Or should it be more proper to do a full COSE translation of RFC3161?